src/cpsr.rs

```diff
@@ -18,6 +18,14 @@ impl Cpsr {
         self.0 & 0x8000 != 0
     }
 
+    pub(crate) fn set_signed(&mut self) {
+        self.0 |= 0b1000_0000_0000_0000_0000_0000_0000_0000;
+    }
+
+    pub(crate) fn set_not_signed(&mut self) {
+        self.0 &= 0b0111_1111_1111_1111_1111_1111_1111_1111;
+    }
```

Comment:
I don't know, probably is more clear `set_sign_flag(true|false)` but I don't have strong opinion now

Reply:
I admit that this choice of mine is based on what I studied on Clean Code ahah

![image](https://user-images.githubusercontent.com/22253036/188737662-f3d9ffbc-e59d-46a8-883b-35d62a0d77bb.png)

Reply:
I'm aware of this but in this case we are treating single bit, we don't handle the boolean in the function for doing different things... But as I wrote, I'm not sure what is the most clean and readable API here